dicomexporter/exporter.py

```diff
@@ -25,12 +48,29 @@ def convertDICOMVolumeToVTKFile(
     """
     Converts DICOM files in a directory into a VTK file (.vti or .vtkjs)
     """
-    # Test output_file_path #
-    if not overwrite and os.path.exists(output_file_path):
-        print('Output file already exist', output_file_path, 
-            '\nIf you want to overwrite the file add the \'--overwrite\' flag')
+    _, file_extensions = extractExtensionsFromFilePath(output_file_path)
+    file_extension = file_extensions[-1] # only handling single file_extensions for now

```

Comment:
What's happening if file_extensions is empty ?

Reply:
Missed that one. Should be fixed now!